files/usr/lib/jeos-firstboot

```diff
@@ -220,6 +226,15 @@ if [ -n "$password" ]; then
     echo "root:$password" | run /usr/sbin/chpasswd
 fi
 
+EFIVARS_DIR="/sys/firmware/efi"
+# Look for EFI dir to see if the machine is booted in UEFI mode
+modprobe efivars
+if ! [ "$(ls -A $EFIVARS_DIR)" ]; then
```

Comment:
Raymund recommends to check for /sys/firmware/efi/systab, that's also what yast does.